felix/bpf/maps.go

```diff
@@ -380,12 +423,21 @@ func (b *PinnedMap) Delete(k []byte) error {
 }
 
 func (b *PinnedMap) updateDeltaEntries() error {
+	logrus.WithField("name", b.Name).Debug("updateDeltaEntries")
+
+	if b.oldfd == b.fd {
+		return fmt.Errorf("old and new maps are the same")
+	}
```

Comment:
QQ: for completeness would it help / add value to add the FD similar to the log entry in L 423?